portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentFolderAPI.macro

```diff
@@ -91,15 +114,7 @@ definition {
 
 	macro assertExternalReferenceCodeWithCorrectValue {
 		if (!(isSet(expectedExternalReferenceCodeValue))) {
-			var folderId = JSONUtil.getWithJSONPath("${responseToParse}", "$.id");
```

Comment:
Moved these into the following method getUuidOfCreatedStructuredContentFolder() so that other test step can call it.